src/App.jsx

```diff
@@ -40,6 +44,7 @@ class App extends Component {
             <Route path="/user" component={User$} />
             <Route path="/holdingpen" component={Holdingpen$} />
             <Route path="/literature" component={Literature$} />
+            <Route component={Errors$} />
```

Comment:
`<Route path="/errors" component={Errors$} />` then fallback  will be a `Redirect` without `from` and `to="/errors"` as we talked.